db/migrate/20200128183708_create_settings.rb

```diff
@@ -5,6 +5,8 @@ def change
       t.string :value
 
       t.timestamps
+
+      t.index :key, unique: true
```

Comment:
why was this index created?

Reply:
this was created because rubocop now fails if there is a `unique` validation in the model and no `unique index` in the migration